src/domain/cards/customerProfileCard/CustomerProfileCard.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FunctionComponent } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Card from '../../../common/card/Card';
+import CardHeader from '../../../common/cardHeader/CardHeader';
+import CardBody from '../../../common/cardBody/CardBody';
+import PrivateCustomerDetails, {
+  PrivateCustomerDetailsProps,
+} from './privateCustomerDetails/PrivateCustomerDetails';
+import OrganizationCustomerDetails, {
+  OrganizationCustomerDetailsProps,
+} from './organizationCustomerDetails/OrganizationCustomerDetails';
+
+export type CustomerProfileCardProps = {
+  className?: string;
+} & (PrivateCustomerDetailsProps | OrganizationCustomerDetailsProps);
+
+const customerHasOrganization = (
+  props: CustomerProfileCardProps
+): props is OrganizationCustomerDetailsProps => {
+  return props.organization !== undefined && props.organization !== null;
```

Comment:
```suggestion
    return (
    (props as OrganizationCustomerDetailsProps).organization !== undefined &&
    (props as OrganizationCustomerDetailsProps).organization !== null
  );
```
according to [type guards](https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards) documentation